src/messaging/ReliableMessageMgr.h

```diff
@@ -102,6 +102,16 @@ class ReliableMessageMgr
      */
     CHIP_ERROR AddToRetransTable(ReliableMessageContext * rc, RetransTableEntry ** rEntry);
 
+    /**
+     *  Calculate the backoff timer for the retransmission.
+     *
+     *  @param[in]   backoffBase    The base interval to use for the backoff calculation, either the active or idle interval.
+     *  @param[in]   sendCount      Count of how many times this message has been sent, including the current attempt.
+     *
+     *  @retval  The backoff time value, including jitter.
+     */
+    System::Clock::Timestamp GetBackoff(System::Clock::Timestamp backoffBase, uint8_t sendCount);
```

Comment:
Does this depend on any member variables? I assume not, in which case we could make this static or not be a part of this class at all and be a free standing function.